### PR TITLE
Release notes for 2.11

### PR DESCRIPTION
**Pull request by dannimad**
## Description

Generate release notes with:

pnpm exec flub generate releaseNotes -g client -t minor --out RELEASE_NOTES/2.11.0.md

### CODE REVIEW

File: RELEASE_NOTES/2.11.0.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+Presence updates are grouped together and throttled to prevent flooding the network with messages when presence values are rapidly updated. This means the presence infrastructure will not immediately broadcast updates but will broadcast them after a configurable delay.
+
+The `allowableUpdateLatencyMs` property configures how long a local update may be delayed under normal circumstances, enabling batching with other updates. The default `allowableUpdateLatencyMs` is **60 milliseconds** but may be (1) specified during configuration of a [States Workspace](https://github.com/microsoft/FluidFramework/tree/main/packages/framework/presence#value-managers#states-workspace) or [Value Manager](https://github.com/microsoft/FluidFramework/tree/main/packages/framework/presence#value-managers#value-managers) and/or (2) updated later using the `controls` member of Workspace or Value Manager. [States Workspace](https://github.com/microsoft/FluidFramework/tree/main/packages/framework/presence#value-managers#states-workspace) configuration applies when a Value Manager does not have its own setting.
```
Comment (tylerbutler) [inline RELEASE_NOTES/2.11.0.md:108]: Links here are a little wrong. I'll clean them up.

File: RELEASE_NOTES/2.11.0.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+
+Commit: [`f7be965`](https://github.com/microsoft/FluidFramework/commit/f7be9651daeba09853627c0953e5969a60674ce3)
+
+Affected packages:
+
+- @fluidframework/presence
+
+[⬆️ Table of contents](#contents)
+
+### Presence updates are now batched and throttled ([#23075](https://github.com/microsoft/FluidFramework/issues/23075))
```
Comment (tylerbutler) [inline RELEASE_NOTES/2.11.0.md:104]: Need to use grouping and batching consistently in this section, preferably picking one or the other. I can fix.

File: RELEASE_NOTES/2.11.0.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+// Configure a states workspace
+const stateWorkspace = presence.getStates(
+  "app:v1states",
+  {
+    // This value manager has an allowable latency of 100ms.
+    position: Latest({ x: 0, y: 0 }, { allowableUpdateLatencyMs: 100 }),
+    // This value manager uses the workspace default.
+    count: Latest({ num: 0 }),
+  },
+  // Specify the default for all value managers in this workspace to 200ms,
```
Comment (tylerbutler) [inline RELEASE_NOTES/2.11.0.md:128]: Use set instead of specify.

File: .changeset/new-hats-learn.md [hunk 4]
```diff
@@ -6,6 +6,9 @@
 "section": other
 ---
 
-Relocating Events Library to `@fluidframework/core-interfaces` and `@fluid-internal/client-utils`
+The events library has been moved from the tree package
 
-The events library's types and interfaces are moved to `@fluidframework/core-interfaces`, while its implementation is relocated to `@fluid-internal/client-utils`. There are no changes to how the events library is used; the relocation simply organizes the library into more appropriate packages. This change has no impact on external consumers of Fluid.
+The tree package contains an events library. The events library's types and interfaces have been moved to
```
Comment (CraigMacomber) [inline .changeset/new-hats-learn.md:11]: "The tree package contains an events library."

No it does not. It used to.

**Review by tylerbutler — COMMENTED** [review]
I noticed a few issues with one of the changesets I added. I'll fix it and open another PR against your fork.

**Review by tylerbutler — APPROVED** [review]
Looks good! Approving for docs.

**Comment by dannimad** [comment]
@microsoft-github-policy-service rerun

**Pull request merged**